wsclient/src/main/java/org/hpccsystems/ws/client/platform/ECLExceptionInfo.java

```diff
@@ -11,6 +12,9 @@
     private String message;
     private String severity;
     private String source;
+    
+    private String audience;
+    private String espCode;

```

Comment:
if we're going to combine the two, why do we distinguish espCode and code (ecl) ? The only new field should be audience.

Reply:
An ECLException object returns an integer code. an ESPException object returns a string code; so they're two different things semantically.  I added a getErrorCode() that returns whichever is set so users wouldn't have to check if they don't want to.

Reply:
I'm not too worried about this one, but espCode in this case is really wswuCode